fdbclient/ClientKnobs.h

```diff
@@ -248,6 +248,9 @@ class ClientKnobs : public KnobsImpl<ClientKnobs> {
 	// blob granules
 	bool ENABLE_BLOB_GRANULES;
 
+	// tenant balancer
+	double REQUEST_TIMEOUT; // rpc request timeout value for tenant balancer clients
```

Comment:
Let's make this name a bit more specific, such as:

```suggestion
	double TENANT_BALANCER_REQUEST_TIMEOUT; // rpc request timeout value for tenant balancer clients
```

Reply:
No problem. Thanks!